packages/ethereum-storage/test/ethereum-blocks.test.ts

```diff
@@ -103,34 +104,30 @@ const mockBlocksEthereum = [
   9807,
   9906,
 ];
+const provider = new providers.JsonRpcProvider('http://localhost:8545');
```

Comment:
We could do this to not depend on any external service for unit tests
(but this would reintroduce some web3 dependencies 😆)
```ts
import ganache from "ganache";
const provider = new ethers.providers.Web3Provider(ganache.provider());
```